### ISSUE TITLE
Add a Help page with more resources

### ISSUE THREAD

**Issue by mssola** [SUSE/Portus#1372]
Right now the Help tab simply redirects users to http://port.us.org, which is fine by now, but we can do better:

- Depends on #1299: add documentation of the API, with a way to test it live.
- We can add man pages from `portusctl`.

**Comment by mssola** [comment]
The help page has now more information, like the API documentation. That being said, we are still missing some elements that might be helpful like man pages for `portusctl`. I'll postpone to 2.5 in the hopes that we can tackle these remaining tasks there.
